src/apps/backend/modules/util/date-utils.ts

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+
+export class DateUtils {
+  static parseDate(dateInput: string | Date): Date {
+    if (typeof dateInput === 'string') {
+      return moment(dateInput, 'YYYY-MM-DD hh:mm A').toDate();
+    }
+    return moment(dateInput).toDate();
```

Comment:
In this method incase of string we are converting it to date else we are just returning the date that we received. Can we refract according to the requirement.